rules/aip0192/no_html.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0192
+
+import (
+	"regexp"
+
+	"github.com/googleapis/api-linter/lint"
+	"github.com/jhump/protoreflect/desc"
+)
+
+var noHTML = &lint.DescriptorRule{
+	Name: lint.NewRuleName(192, "no-html"),
+	LintDescriptor: func(d desc.Descriptor) []lint.Problem {
+		for _, comment := range separateInternalComments(d.GetSourceInfo().GetLeadingComments()).External {
```

Comment:
Add a `OnlyIf` `d.GetSourceInfo() != nil`.

Reply:
Good idea. That also requires adding `OnlyIf` to `DescriptorRule`. Done.